runtime/jcl/common/sun_misc_Unsafe.cpp

```diff
@@ -1101,7 +1101,7 @@ Java_jdk_internal_misc_Unsafe_registerNatives(JNIEnv *env, jclass clazz)
 	if (J2SE_VERSION(currentThread->javaVM) >= J2SE_V11) {
 		registerJdkInternalMiscUnsafeNativesJava10(env, clazz);
 	}
-	if (J2SE_VERSION(currentThread->javaVM) >= J2SE_V14) {
+	if (J2SE_VERSION(currentThread->javaVM) > J2SE_V11) {
```

Comment:
I think you should leave things like this alone. If we decide to clean up, it should be done separately.

Reply:
I feel these obsoleted defines don't have much references, could be replaced in a straightforward way for cleaner code.
If you prefer a separated PR for such changes, will open one then.

Reply:
Created https://github.com/eclipse-openj9/openj9/pull/12935